rocketpy/AeroSurface.py

```diff
@@ -1892,53 +1885,113 @@ def allInfo(self):
-        lower_button_position : int, float
-            Position of the lower rail button in meters. The lower button is the one
-            closest to the nose cone and furthest from the tail. The coordinate system
-            used is the same as the Rocket that the buttons will be a part of.
+        buttons_distance : int, float
+            Distance between the first and the last rail button in meters.
         angular_position : int, float, optional
             Angular position of the rail buttons in degrees measured
             as the rotation around the symmetry axis of the rocket
             relative to one of the other principal axis.
+        name : string, optional
+            Name of the rail buttons. Defaul is "Rail Buttons".
 
         Returns
         -------
         None
 
         """
-        self.upper_button_position = upper_button_position
-        self.lower_button_position = lower_button_position
+        self.buttons_distance = buttons_distance
         self.angular_position = angular_position
+        self.name = name
+
+        self.evaluateLiftCoefficient()
+        self.evaluateCenterOfPressure()
+        return None
+
+    def evaluateCenterOfPressure(self):
+        """Evaluates the center of pressure of the rail buttons. Rail buttons
+        do not contribute to the center of pressure of the rocket.
+
+        Returns
+        -------
+        None
+        """
+        self.cpx = 0
+        self.cpy = 0
+        self.cpz = 0
+        self.cp = (self.cpx, self.cpy, self.cpz)
+        return None
+
+    def evaluateLiftCoefficient(self):
+        """Evaluates the lift coefficient curve of the rail buttons. Rail
+        buttons do not contribute to the lift coefficient of the rocket.
+
+        Returns
+        -------
+        None
+        """
+        self.clalpha = Function(
+            lambda mach: 0,
+            "Mach",
+            f"Lift coefficient derivative for {self.name}",
+        )
+        self.cl = Function(
+            lambda alpha, mach: 0,
+            ["Alpha (rad)", "Mach"],
+            "Cl",
+        )
+
+    def evaluateGeometricalParameters(self):
+        """Evaluates the geometrical parameters of the rail buttons. Rail
+        buttons do not contribute to the geometrical parameters of the rocket.
+
+        Returns
+        -------
+        None
+        """
+        return None
+
+    def geometricalInfo(self):
+        """Returns the geometrical info of the rail buttons. Rail buttons
+        do not have geometrical parameters.
+
+        Returns
+        -------
+        None
+        """
+        return None
+
+    def aerodynamicInfo(self):
+        """Returns the aerodynamic info of the aerodynamic surface.
+
+        Returns
+        -------
+        None
+        """
         return None
 
-    def __repr__(self):
-        rep = f"Rail buttons pair at positions {self.upper_button_position} m and {self.lower_button_position} m"
-        return rep
```

Comment:
It might be a commentary regarding `__repr__` more generally for the AeroSurfaces PR. However, why to remove the `__repr__` (and `__str__`)? I understand that there is already plenty of info in `allInfo`, however the `__repr__` is quite important to make sense of the object in a compact way, for instance mentioning the `self.name` attribute.